src/expandableCalendar/style.js

```diff
@@ -1,13 +1,13 @@
 import {StyleSheet, Platform} from 'react-native';
 import * as defaultStyle from '../style';
-
+import {HEADER_HEIGHT} from './index.js';
```

Comment:
importing from index can create cyclic require. 
Consider moving the constants to a 3rd file, or make sure the imports are in one direction (maybe declare it in styles and import it from there in the index)

Reply:
Nice! Done